### DISCUSSION TITLE
Icons on top-level navigation bar?

### DISCUSSION THREAD

**Discussion by YozoraXCII** [squidfunk/mkdocs-material#6581] [Q&A] [answered]
Looking for a quick bit of advice.

I'm trying to figure out how to add icons to the navigation header items, as can be seen here:

https://docs.kolena.io/

I can get it to individual pages using the following:

```
---
icon: material/bookshelf
---
```

but if I want to add the icon to the top-level navigation which I am defining in mkdocs.yml, if I add `:fontawesome-brands-docker:` then I get mapping errors due to the colon, if I wrap it in quotation marks my title becomes `:fontawesome-brands-docker: Docker`

Thanks in advance for any assistance

**Comment by kamilkrzyskow** [comment, marked as answer]
You can't assign icons to sections without index pages. 
On the linked site you can see that the "navigation header items" can be clicked and they point to a rendered page.
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages
https://github.com/kolenaIO/kolena/blob/trunk/docs/index.md?plain=1

If you don't want to use index pages you would have to override the template files and write your own logic to input the icon next to the title.
https://github.com/squidfunk/mkdocs-material/discussions/6419#discussioncomment-7745273
